diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-3.0.0/root/etc/e-smith/events/actions/turba_upgrade mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/events/actions/turba_upgrade
--- e-smith-turba-3.0.0/root/etc/e-smith/events/actions/turba_upgrade	2009-10-08 21:43:25.000000000 -0500
+++ mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/events/actions/turba_upgrade	2009-10-08 21:42:14.000000000 -0500
@@ -50,23 +50,19 @@
 #and on a 6.x and 7.x install, this would have already been run.  John H. Bennett III
 # These are all safe to run multiple times
 
-#my @statements = (
-#    'CHANGE object_homeAddress object_homeaddress VARCHAR(255)',
-#    'CHANGE object_workAddress object_workaddress VARCHAR(255)',
-#    'CHANGE object_homePhone object_homephone VARCHAR(25)',
-#    'CHANGE object_workPhone object_workphone VARCHAR(25)',
-#    'CHANGE object_cellPhone object_cellphone VARCHAR(25)',
-#    'MODIFY object_title VARCHAR(255)',
-#    'MODIFY object_company VARCHAR(255)',
-#);
-
-#foreach my $statement (@statements)
-#{
-#    $statement =
-#        $db_turbahandle->prepare("alter table turba_objects $statement")
-#            or die "prepare: $$statement: $DBI::errstr";
-#    $statement->execute or die "execute: $$statement: $DBI::errstr";
-#}
+my @statements = (
+     "ALTER TABLE turba_shares CHANGE share_owner share_owner VARCHAR(255)",
+     "ALTER TABLE turba_shares_users CHANGE user_uid user_uid VARCHAR(255)",
+     "ALTER TABLE turba_shares_groups CHANGE group_uid group_uid VARCHAR(255)",
+);
+  
+foreach my $statement (@statements)
+{
+    $statement =
+        $db_turbahandle->prepare("$statement")
+            or die "prepare: $$statement: $DBI::errstr";
+    $statement->execute or die "execute: $$statement: $DBI::errstr";
+}
 
 # We now need to create some columns, but we need to first check
 # whether they exist already
@@ -447,36 +443,6 @@
     $statement->execute or die "execute: $$statement: $DBI::errstr";
 }
 
-    unless (defined $turba_shares->{share_owner})
-{
-    my $statement =
-        "ALTER TABLE turba_shares CHANGE share_owner share_onwer ".
-        "VARCHAR(255)";
-    $statement = $db_turbahandle->prepare($statement) or
-        die "prepare: $$statement: $DBI::errstr";
-    $statement->execute or die "execute: $$statement: $DBI::errstr";
-}
-
-    unless (defined $turba_shares_users->{user_uid})
-{
-    my $statement =
-        "ALTER TABLE turba_shares_users CHANGE user_uid user_uid ".
-        "VARCHAR(255)";
-    $statement = $db_turbahandle->prepare($statement) or
-        die "prepare: $$statement: $DBI::errstr";
-    $statement->execute or die "execute: $$statement: $DBI::errstr";
-}
-
-    unless (defined $turba_shares_groups->{group_uid})
-{
-    my $statement =
-        "ALTER TABLE turba_shares_groups CHANGE group_uid group_uid ".
-        "VARCHAR(255)";
-    $statement = $db_turbahandle->prepare($statement) or
-        die "prepare: $$statement: $DBI::errstr";
-    $statement->execute or die "execute: $$statement: $DBI::errstr";
-}
-
 # Create an index for owner_id if needed
 unless ($turba_objects->{owner_id}->{Key})
 {
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-3.0.0/root/etc/e-smith/templates/etc/openldap/slapd.conf/12hordeschema mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/etc/openldap/slapd.conf/12hordeschema
--- e-smith-turba-3.0.0/root/etc/e-smith/templates/etc/openldap/slapd.conf/12hordeschema	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/etc/openldap/slapd.conf/12hordeschema	2009-10-08 21:05:57.000000000 -0500
@@ -0,0 +1 @@
++include         /etc/openldap/schema/horde.schema
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/00header mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/00header
--- e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/00header	2009-10-08 21:43:25.000000000 -0500
+++ mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/00header	2009-10-08 21:06:36.000000000 -0500
@@ -1,4 +1,4 @@
 //00header
 /**
- * $Horde: turba/config/sources.php.dist,v 1.97.6.38 2008/11/12 06:29:26 wrobel Exp $
+ * $Horde: turba/config/sources.php.dist,v 1.97.6.41 2009/08/05 21:06:10 jan Exp $  
  */
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP
--- e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP	2008-04-22 01:19:26.000000000 -0500
+++ mezzanine_patched_e-smith-turba-3.0.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP	2009-10-08 21:17:49.000000000 -0500
@@ -17,22 +17,41 @@
          'port' => 389,
          'tls' => false,
 {
-    my $domainName = $DomainName;
-    $domainName =~ s/\./,dc=/g;
-    $OUT = qq(        'root' => 'dc=$domainName',);
+   #my $smeversion = $sysconfig{ReleaseVersion} || "";  
+   my $domainName = $DomainName;
+   $domainName =~ s/\./,dc=/g;
+   #
+   #if  ($smeversion < '8')
+   #{
+      $OUT = qq(        'root' => 'dc=$domainName',);
+   #} else {
+   #   $OUT = qq(         'root' => 'ou=Users,dc=$domainName',);
+   #}
 }
 //         'bind_dn' => 'cn=admin,ou=users,dc=example,dc=com',
+//         // For Active Directory:   
+//         // 'bind_dn' => 'username@example.com',
 //         'bind_password' => '********',
          'sizelimit' => 200,
+//         // For Active Directory:  
+//         // 'sizelimit' => 0,
          'dn' => array('cn'),
          'objectclass' => array('top',
                                 'person',
                                 'organizationalPerson',
                                 'inetOrgPerson',
                                 'turbaContact'),
-								  // Add 'turbaContact' to this array if using
-                                // 'turbaType' attribute below
+                                // Add 'turbaContact' to this array if using
+                                // 'turbaType' attribute below, and 'calEntry'
+//                              // if using 'freebusyUrl'.
+//         // For Active Directory:
+//         // 'objectclass' => array('organizationalPerson',
+//         //                        'user',
+//         //                        'group',
+//         //                        'contact'),
          'scope' => 'one',
+//         // For Active Directory:
+//         // 'scope' => 'sub',
          'charset' => 'UTF-8',
 //         // Consult the LDAP schema to verify that all required attributes for
 //         // an entry are set and add them if needed.
@@ -43,10 +62,19 @@
 //         // field is assumed to have postalAddress syntax; otherwise the schema
 //         // is consulted for the syntax to use.
 //         'checksyntax' => false,
-         'version' => 3
+         'version' => 3,
+//
+//         // For Active Directory you probably want to also set the following
+//         // parameters:
+//         // 'deref' => LDAP_DEREF_ALWAYS,
+//         // 'filter' => '&(SAMAccountName=*)(mail=*)',
+//         // 'referrals' => 0,
      ),
      'map' => array(
          '__key' => 'dn',
+
+//         // Remove this mapping if using Active Directory server:
+//         '__uid' => 'uid',
          '__uid' => 'uid',
 
 //         // From horde.schema.  Make sure you have 'turbaContact' objectClass
@@ -59,14 +87,20 @@
          'homePhone' => 'homephone',
          'workPhone' => 'telephonenumber',
          'cellPhone' => 'mobiletelephonenumber',
-         'homeAddress' => 'homepostaladdress',
+         'homeAddress' => 'street',
 //
 //         // From rfc2739.schema:
 {
         my $hordefreebusy = $horde{'freebusy'} || 'disabled'; return "        #freebusy is currently not enabled.  To enable - config setprop horde freebusy enabled" unless ($hordefreebusy eq "enabled");
         $OUT .= '        \'freebusyUrl\' => \'calFBURL\'';
 }
-//         // 'freebusyUrl' => 'calFBURL',
+//
+//         // For Active Directory servers:
+//         // 'name' => 'displayname',
+//         // 'title' => 'title',
+//         // 'cellPhone' => 'mobile',
+//         // 'department' => 'department',
+//         // 'company' => 'company',
      ),
      'search' => array(
          'name',
@@ -82,6 +116,11 @@
      'approximate' => array(
          'cn',
      ),
+//     // For Active Directory servers:
+//     // 'approximate' => array(
+//     //     'displayname',
+//     //     'samaccountname',
+//     // ),
      'export' => true,
      'browse' => true,
 );
