--- e-smith-turba-2.2/createlinks.migrate_turba_contacts	2008-06-20 22:34:18.000000000 -0500
+++ e-smith-turba-2.2/createlinks	2008-07-11 20:53:16.000000000 -0500
@@ -38,6 +38,18 @@
 
 foreach (qw(
     40mysql.create.turba
+    50turba_upgrade
+    57turba_mysql_reset_addressbook_pref
+    60migrate-imp-to-turba
+    ))
+{
+    templates2events(
+	"/etc/e-smith/sql/init/$_",
+	qw(post-install));
+}
+
+foreach (qw(
+    40mysql.create.turba
     49turba_migrate_contacts
     50turba_upgrade
     57turba_mysql_reset_addressbook_pref
@@ -46,7 +58,7 @@
 {
     templates2events(
 	"/etc/e-smith/sql/init/$_",
-	qw(post-install post-upgrade));
+	qw(post-upgrade));
 }
 
 
