diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP	2007-04-26 19:59:20.000000000 -0600
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/sources.php/50LocalLDAP	2007-04-26 19:58:52.000000000 -0600
@@ -8,7 +8,11 @@
         'server' => 'localhost',
         'port' => 389,
         'tls' => false,
-        'root' => '',
+{
+    my $domainName = $DomainName;
+    $domainName =~ s/\./,dc=/g;
+    $OUT = qq(        'root' => 'dc=$domainName',);
+}
         'sizelimit' => 200,
         'dn' => array('cn'),
         'objectclass' => array('top',
