diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf mezzanine_patched_e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf
--- e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-02-26 14:00:24.293002721 +1100
+++ mezzanine_patched_e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-02-26 13:54:36.210565154 +1100
@@ -1,5 +1,4 @@
 // 100Conf
 $conf['menu']['import_export'] = true; 
-$conf['menu']['apps'] = array('imp');
 $conf['client']['addressbook'] = 'localsql';
 $conf['search']['cache'] = true;
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings mezzanine_patched_e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings
--- e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-turba-1.5.1/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings	2006-02-26 13:55:34.762377796 +1100
@@ -0,0 +1,28 @@
+//120MenuSettings
+{
+$OUT = '$conf[\'menu\'][\'apps\'] = array(';
+
+use esmith::ConfigDB;
+my $c = esmith::ConfigDB->open_ro;
+
+my $record = $c->get('horde');
+
+foreach $component (qw/ accounts agora chora forwards genie giapeta goops gollem hermes ingo imp
+    jeta jonah juno kronolith klutz luxor merk midas mimp mnemo mottle nag occam orator passwd
+    rakim sam scry sork swoosh trean thor troll turba vacation ulaform vilma whups wicked /) {
+
+    my $status = $record->prop($component);
+
+    if (!defined ($status)) {
+        $status = '';
+    }
+
+    if ($status eq 'installed') {
+        $OUT .= '\'' . $component . '\', '
+    }
+}
+
+$OUT = substr($OUT, 0, length($OUT) - 2) . ");\n";
+
+}
+
