diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppTurba mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppTurba
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppTurba	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppTurba	2006-12-22 09:13:18.000000000 -0700
@@ -0,0 +1,5 @@
+{
+my $TurbaMenu = $turba{MenuArray} || "enabled"; return "" unless ($TurbaMenu eq "enabled");
+$apps{turba} = 1;
+$OUT = '';
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-12-22 09:14:28.000000000 -0700
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-12-22 09:10:24.000000000 -0700
@@ -1,6 +1,5 @@
 // 100Conf
 $conf['menu']['import_export'] = true; 
-$conf['menu']['apps'] = array('imp');
 $conf['client']['addressbook'] = 'localsql';
 $conf['shares']['source'] = 'sql';
 $conf['comments']['allow'] = true;
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/120MenuSettings	2006-12-22 09:11:36.000000000 -0700
@@ -0,0 +1,2 @@
+// 120MenuSettings
+include '/home/httpd/html/horde/conf.menu.apps.php';
