diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/00header mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/00header
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/00header	2006-03-15 20:48:40.000000000 -0600
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/00header	2006-09-20 23:32:38.000000000 -0500
@@ -1,2 +1,2 @@
 /* CONFIG START. DO NOT CHANGE ANYTHING IN OR AFTER THIS LINE. */
-// $Horde: turba/config/conf.xml,v 1.6 2004/12/08 17:08:37 jan Exp $
+// $Horde: turba/config/conf.xml,v 1.6.2.3 2006/05/09 15:58:10 jan Exp $
diff -Nur -x '*.orig' -x '*.rej' e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf
--- e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-03-15 20:48:40.000000000 -0600
+++ mezzanine_patched_e-smith-turba-1.7.0/root/etc/e-smith/templates/home/httpd/html/horde/turba/config/conf.php/100Conf	2006-09-20 23:33:27.000000000 -0500
@@ -2,4 +2,6 @@
 $conf['menu']['import_export'] = true; 
 $conf['menu']['apps'] = array('imp');
 $conf['client']['addressbook'] = 'localsql';
-$conf['search']['cache'] = true;
+$conf['shares']['source'] = 'sql';
+$conf['comments']['allow'] = true;
+$conf['documents']['type'] = 'horde';
